pdns/ueberbackend.cc

```diff
@@ -273,16 +280,36 @@ void UeberBackend::reload()
   }
 }
 
+void UeberBackend::updateDomainCache() {
+  if (!g_domainCache.isEnabled()) {
+    return;
+  }
+
+  vector<tuple<DNSName, int>> domain_indices;
+  g_domainCache.setReplacePending();
+
+  for (vector<DNSBackend*>::iterator i = backends.begin(); i != backends.end(); ++i )
+  {
+    vector<DomainInfo> domains;
+    (*i)->getAllDomains(&domains, false);
```

Comment:
Please include disabled domains as well, otherwise re-enabled zones and new secondary zones will not show up until the domain cache has expired. This will prevent the use of large ttl's on secondaries and api only installs.

Reply:
My opinion: it's perfectly fine that disabled zones are not visible until the cache expires.

Reply:
What is the advantage of not including disabled zones? 
Performance comes to mind, but the performance impact of including disabled zones is close to zero (the extra required SOA lookup is cached by the query cache).
For new secondary zones (zones without records) fast availability of new zone content is important. Some registries have a predelegation checks (.de). Not including disabled zones here will delay this check for no reason and will reduce the usefulness of this cache.

Reply:
The original usecase doesn't really care about either. I'll flip this to true, but I can't do much thinking what impact it will have.